examples/simple/package.json

```diff
@@ -7,7 +7,6 @@
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-plotly.js": "^1.0.4",
-    "react-plotly.js-editor": "0.2.0-alpha.1",
```

Comment:
why does the `simple` app not get its own editor but the others do?

Reply:
oh, I think this must have happened when npm link was ran.

I added an `npm setup` script to the editor that does the npm linking of the editor to the simple app, 

so that when someone clones the editor and just wants to see how it works or get up and running developing, they can just clone, npm setup, npm start, like Ben did in his batch repo

it's when the setup script runs that it must have been removed, I'll add it back

Reply:
OK. We could also have the setup script not do the `npm link` dance, though, right?

Reply:
yeah, i could remove that